src/libslic3r/PrintObject.cpp

```diff
@@ -428,6 +432,25 @@ void PrintObject::generate_support_material()
     }
 }
 
+void PrintObject::prepare_adaptive_infill_data()
+{
+    float fill_density = this->print()->full_print_config().opt_float("fill_density");
+    float infill_extrusion_width = this->print()->full_print_config().opt_float("infill_extrusion_width");
+
+    coordf_t line_spacing = infill_extrusion_width / ((fill_density / 100.0f) * 0.333333333f);
+
+    BoundingBoxf bed_shape(this->print()->config().bed_shape.values);
+    BoundingBoxf3 printer_volume(Vec3d(bed_shape.min(0), bed_shape.min(1), 0),
```

Comment:
Please use bed_shape.min.x() instead of min(0) for better readibility, if the Eigen vectors represent 2D or 3D vectors.